aio_pika/connection.py

```diff
@@ -201,8 +204,7 @@ async def ready(self) -> None:
     def __del__(self) -> None:
         if (
             self.is_closed or
-            self.loop.is_closed() or
-            not hasattr(self, "connection")
```

Comment:
If I overlooked some method how `self` could end up with an `connection` attribute I will undo this change.